v1.1: Update index size/count limits (#2224)
meilisearch/documentation: learn/advanced/known_limitations.md

## Maximum database size

> **Limitation:** The maximum index size Meilisearch supports on Linux environments is around 80TiB. For performance reasons, Meilisearch recommends keeping indexes under 2TiB.

Reviewer: ```suggestion
**Limitation:** Meilisearch supports a maximum index size of around 80TiB on Linux environments.  For performance reasons, Meilisearch recommends keeping indexes under 2TiB.
```
You can ignore this one but I thought it was easier to read

> **Explanation:** Meilisearch can accommodate indexes of any size as long the combined size of active databases is below the maximum virtual address space the OS devotes to a single process. On Linux, this limit is approximmately 80TiB.

Reviewer 2: ```suggestion
**Explanation:** Meilisearch can accommodate indexes of any size as long the combined size of active databases is below the maximum virtual address space the OS devotes to a single process. On Linux, this limit is approximately 80TiB.
```

Author: Dishonour on my family! Thank you for catching that typo 👍

> **Explanation:** Meilisearch can accommodate indexes of any size as long the combined size of active databases is below the maximum virtual address space the OS devotes to a single process. On Linux, this limit is approximmately 80TiB.

Reviewer: Should we be specific and mention x64 Linux?
```suggestion
**Explanation:** Meilisearch can accommodate indexes of any size as long the combined size of active databases is below the maximum virtual address space the OS devotes to a single process. On x64 Linux, this limit is approximately 80TiB.
```

Reviewer 2: If we do, I'd prefer we use the expression `64-bit Linux` then because `x64` is not the formal name of the architecture (which is `amd64`), and the limit should be the same for non-Intel (e.g., `aarch64` (ARM)) architectures.
